nvtabular/framework_utils/tensorflow/layers/embedding.py

```diff
@@ -260,14 +308,18 @@ class ScalarLinearFeatures(tf.keras.layers.Layer):
     ----------
     feature_columns : list of tf.feature_column
         feature columns describing the inputs to the layer
+    sparse_aggregation : str in ("mean", "sqrtn", "sum")
+        how to aggregate embeddings from multi-valent categorical features.
+        See `tf.nn.embedding_lookup_sparse` documentation for details on
+        how this is used.
```

Comment:
Is this sparse aggregation parameter used (in the base class?). 

I'd prefer not to rely on kwargs for parameters lists when possible, can we add this parameter to the list of params?

Reply:
No I added this kwarg temporarily but removed it, just pushed a change to remove it from the doc too